chrome_48.txt

```diff
@@ -0,0 +1,19 @@
+[Section]
+Name = Chrome 48
+Version = 48.0.2564.109
+License = Freeware
+Description = Google Chrome is a cross-platform web browser developed by Google, based on the open source base of Chromium.
+Category = 5
+URLSite = https://www.google.com/intl/es_es/chrome/
+URLDownload = https://archive.org/download/48.0.2564.109-chrome-installer/48.0.2564.109_chrome_installer_win32.exe
```

Comment:
What is the purpse of having a 32 bit Chrome 48? Does it do anything better than the 32bit Chromium-XP versions that we do have in rapps already? (sorry for the question, I do appreciate all the hard work that you put into this PR)

Reply:
Just to have the same version in both architectures (32 and 64 bits in parallel) only for comparison/testing purposes.
And well: Better to sync than Chromium if you are using G services. But I can delete it if anybody finds a amd64 version of your Chromium, that would be amazing.